Microsoft/WebTemplateStudio
# Flask - Function and Variable names should reflect the PEP8 Python Standards

Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/29399

**Changes**
- Adapt all the functions and variables in Flask templates to respect PEP8 standards

**Note**: Please look for any places that I might have missed to incorporate similar standards.

**How to Test**
- This PR touches many Flask templates. So, the best way to test this is to generate all 3 types of projects (with Flask as the backend framework):
    - All pages without any CosmosDB
    - All pages with CosmosDB Mongo
    - All pages with CosmosDB SQL
- Try to run the project and it should work appropriately.

closes #785
